src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,14 @@
 // Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Prop = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Prop> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (<TodoInfo todo={todo} key={todo.id} />
+    ))}
```

Comment:
```suggestion
    {todos.map(todo => (
      <TodoInfo todo={todo} key={todo.id} />
    ))}
```
```
( somecode
)
```
is a bit strange formatting 😉

Reply:
Fixed!